### ISSUE TITLE
Expected an ARRAY reference to zip

### ISSUE THREAD

**Issue by eserte** [robrwo/Const-Exporter#15]
Some tests started to fail:
```
Expected an ARRAY reference to zip at /usr/home/cpansand/.cpan/build/2021040604/Const-Exporter-v1.1.2-0/blib/lib/Const/Exporter.pm line 79.
BEGIN failed--compilation aborted at t/10-basic.t line 33.
t/10-basic.t ................ 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
Expected an ARRAY reference to zip at /usr/home/cpansand/.cpan/build/2021040604/Const-Exporter-v1.1.2-0/blib/lib/Const/Exporter.pm line 79.
BEGIN failed--compilation aborted at t/21-example.t line 10.
t/21-example.t .............. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
Expected an ARRAY reference to zip at /usr/home/cpansand/.cpan/build/2021040604/Const-Exporter-v1.1.2-0/blib/lib/Const/Exporter.pm line 79.
BEGIN failed--compilation aborted at t/lib/Test/Const/Exporter/Enums.pm line 15.
Compilation failed in require at t/enums.t line 7.
BEGIN failed--compilation aborted at t/enums.t line 7.
t/enums.t ................... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
```

Maybe caused by `List::AllUtils` or one of its `List::*` dependencies?

**Comment by eserte** [comment]
`List::Util` added a `zip` function in its latest release, so this might cause the problem.

**Comment by robrwo** [comment]
Should be fixed in v1.2.0
